bindings/nodejs/generated/main.cc.prologue

```diff
@@ -8,5 +9,6 @@ using namespace v8;
 #include "../src/common.h"
 
 NAN_MODULE_INIT(Init) {
+  sol_init();
```

Comment:
Not sure this is the best path to go. But let's digest in parts:

 - what you mean "polling for hangup"? I'm not sure this is UNIX signals, children PID (ie: waitpid) or something else. If you want to monitor for signals you could use `signalfd()`, or just ignore it.
 - what happens in node.js/libuv if you have no file descriptors to monitor but you still have idlers and timers? If it still works, than that's fine. Soletta not quitting if there is nothing to do (ie: no fds, no idlers, no timers) is me being lazy to implement. I could  "justify" that we still want to listen for signals, but that's not very useful -- would be just an excuse.

your current implementation seems to work, looks like you have tested it, but I'm afraid it would cause us problems with us changing the expectations of nodejs users.